frontend_server/lib/server.dart

```diff
@@ -388,15 +77,14 @@ Future<int> starter(
     final Directory temp = Directory.systemTemp.createTempSync('train_frontend_server');
     try {
       final String outputTrainingDill = path.join(temp.path, 'app.dill');
-      options =
-          _argParser.parse(<String>['--incremental', '--sdk-root=$sdkRoot',
-          '--output-dill=$outputTrainingDill'
-          ]);
-      compiler ??=
-      new _FrontendCompiler(output, printerFactory: binaryPrinterFactory);
-
-      await compiler.compile(
-          Platform.script.toFilePath(), options, generator: generator);
+      options = frontend.argParser.parse(<String>[
+        '--incremental',
+        '--sdk-root=$sdkRoot',
+        '--output-dill=$outputTrainingDill',
+        '--target=flutter']);
+      compiler ??= new _FlutterFrontendCompiler(output, trackWidgetCreation: true);
```

Comment:
Previously, trackWidgetCreation was not enabled in a training run. Is it an intentional change?

Reply:
Yes, I thought it make sense to train with trackWidgetCreation enabled.